### PR TITLE
TDI-38333:Make i18n for Azure Blob, Azure queue and Azure table

### PR DESCRIPTION
**Pull request by pyzhou-talend**
https://jira.talendforge.org/browse/TDI-38333
**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/AzureStorageQueueDeleteReader.java [hunk 1]
```diff
@@ -28,11 +28,14 @@
 import org.talend.components.api.container.RuntimeContainer;
 import org.talend.components.api.exception.ComponentException;
 import org.talend.components.api.properties.ComponentProperties;
+import org.talend.components.azurestorage.blob.runtime.AzureStorageDeleteReader;
```
Comment (pteyssier) [inline components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/AzureStorageQueueDeleteReader.java]: wrong import here

File: components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/AzureStorageQueueInputLoopReader.java [hunk 2]
```diff
@@ -24,7 +24,10 @@
 import org.talend.components.api.component.runtime.BoundedSource;
 import org.talend.components.api.container.RuntimeContainer;
 import org.talend.components.api.exception.ComponentException;
+import org.talend.components.azurestorage.blob.runtime.AzureStorageDeleteReader;
```
Comment (pteyssier) [inline components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/AzureStorageQueueInputLoopReader.java]: same here

File: components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/AzureStorageQueueInputReader.java [hunk 3]
```diff
@@ -28,9 +28,12 @@
 import org.talend.components.api.component.runtime.BoundedSource;
 import org.talend.components.api.container.RuntimeContainer;
 import org.talend.components.api.exception.ComponentException;
+import org.talend.components.azurestorage.blob.runtime.AzureStorageDeleteReader;
```
Comment (pteyssier) [inline components/components-azurestorage/src/main/java/org/talend/components/azurestorage/queue/runtime/AzureStorageQueueInputReader.java]: ...

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/305/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/305/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/308/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/308/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/313/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/313/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/314/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/314/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/315/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/315/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/326/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/326/)

**Pull request merged**